ambari-server/src/main/java/org/apache/ambari/server/state/ServiceGroupFactory.java

```diff
@@ -29,6 +29,7 @@
 
   ServiceGroup createNew(Cluster cluster,
                          @Assisted("serviceGroupName") String serviceGroupName,
+                         @Assisted("stackId") String stackId,
```

Comment:
stackId should be the object StackId in this method

Reply:
+1 for `StackId`

Reply:
I will change to stackId instance